core/src/main/scala/org/apache/spark/api/r/RUtils.scala

```diff
@@ -23,6 +23,10 @@ import java.util.Arrays
 import org.apache.spark.{SparkEnv, SparkException}
 
 private[spark] object RUtils {
+  // Local path where R binary packages built from R source code contained in the spark
+  // packages specified with "--packages" or "--jars" command line option reside.
+  var rPackages: Option[String] = None
```

Comment:
I find this hard to maintain. Where `RUtils.rPackages` is called would change from deployment type, right? The path will change from cluster mode to client, right?